### PR TITLE
add landing message. #1

### PR DESCRIPTION
**Pull request by anjiin-sama**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <title>Welcome to Potionify</title>
 </head>
 <body>
-    
+    <h1>Welcome to Potionify! Enjoy your stay.</h1>
```
Comment (QualitatsKontrolle) [inline index.html]: could you remove the 'Enjoy your stay' string, and make the 'Welcome to Potionify' string red?

Comment (anjiin-sama) [reply index.html]: sure, can do!

**Review by QualitatsKontrolle — CHANGES_REQUESTED** [review]
Change the code as per my comments on the review, kindly

**Review by QualitatsKontrolle — APPROVED** [review]
Well done, thank you!

**Pull request merged**